tools/run_tests/run_tests_matrix.py

```diff
@@ -262,6 +272,18 @@ def _allowed_labels():
     print '  %s' % job.shortname
 print
 
+if args.filter_pr_tests:
+  print 'IMPORTANT: Test filtering is not active; this is only for testing.'
+  relevant_jobs = filter_tests(jobs, args.base_branch)
+  print
+  if len(relevant_jobs) == len(jobs):
+    print '(TESTING) No tests will be skipped.'
+  else:
+    print '(TESTING) These tests will be skipped:'
+    for job in list(set(jobs) - set(relevant_jobs)):
```

Comment:
as a TODO we could add skipped tests to the report.xml, because then it's going to be obvious from Jenkins UI what has been skipped.